environment/scripts/prepare.sh

```diff
@@ -38,6 +40,10 @@ while [[ $# -gt 0 ]] ; do
       LOCAL_CORE_SRC=$2
       shift 2
       ;;
+		--nearup_source)
```

Comment:
What is the value of passing info via arguments instead of passing it through evn variables?